app/livechat/client/views/sideNav/livechat.js

```diff
@@ -132,11 +134,21 @@ Template.livechat.onCreated(function() {
 			this.statusLivechat.set();
 		}
 	});
-
-	this.subscribe('livechat:inquiry');
-
-	this.updateAgentDepartments = (/* data */) => {
-		// TODO: reload agent departments and the inquiries..
+	if (!settings.get('Livechat_enable_inquiry_fetch_by_stream')) {
```

Comment:
Why not implement this code in a helper method instead of directly on the template?